notebooks/kepler/kepler.jl

```diff
@@ -0,0 +1,63 @@
+
+# Set up required packages
+using Pkg
+Pkg.activate(@__DIR__)
+Pkg.instantiate()
+
+
+# Load packages
+using BSeries
+using SymPy
+using Latexify
+using LinearAlgebra
+
+# Kepler problem
+function f_p(q, p)
+  (-1 / sqrt(q[1]^2 + q[2]^2)^3) .* q
+end
+
+function f_q(q, p)
+  p
+end
+
+# Störmer-Verlet method as additive RK method, see
+# Hairer, Lubich, Wanner (2002)
+# Geometric numerical integration
+# Table II.2.1
+As = [
+    [0 0; 1//2 1//2],
+    [1//2 0; 1//2 0],
+]
+bs = [
+    [1//2, 1//2],
+    [1//2, 1//2]
+]
+ark = AdditiveRungeKuttaMethod(As, bs)
+
+# Set up symbolic equation
+dt_sym = symbols("h", real=true)
+q_sym = symbols("q_1, q_2", real=true)
+p_sym = symbols("p_1, p_2", real=true)
+u_sym = [q_sym..., p_sym...]
+f_q_sym = [f_q(q_sym, p_sym)..., 0, 0]
+f_p_sym = [0, 0, f_p(q_sym, p_sym)...]
+f_sym = (f_q_sym, f_p_sym)
+
+# Compute B-series of the numerical integrator and the modified equation
+truncation_order = 8
+series_integrator = bseries(ark, truncation_order)
+series = modified_equation(f_sym, u_sym, dt_sym, series_integrator);
+
+# check that it contains no terms proportional to odd powers of h:
+println("Modified equation terms with odd powers of h:")
+expr = SymPy.collect(series[1],dt_sym)
+for j in 0:3
+    println(expr.coeff(dt_sym,2*j+1), " ", dt_sym^(2*j+1))
+end
+
+# Compute B-series of the numerical integrator and the modifying integrator
+truncation_order = 4
+series = modifying_integrator(f_sym, u_sym, dt_sym, series_integrator)
```

Comment:
```suggestion
truncation_order = 4
series_integrator = bseries(ark, truncation_order)
series = modifying_integrator(f_sym, u_sym, dt_sym, series_integrator)
```
To only print the modifying integrator stuff up to order 4